setup.py

```diff
@@ -132,6 +139,8 @@ def get_cmake_args():
 
     if sys.platform != 'win32':
         os.environ['SKBUILD_BUILD_OPTIONS'] = f'-j{num_threads}'
+    elif use_msbuild:
+        os.environ['SKBUILD_BUILD_OPTIONS'] = f'/M'
```

Comment:
Can you leave a comment say `/M` is used for multi-thread compilation

Reply:
So this is an example, there's no "user" for this code. As for which feature it's using, this code is using designated initializers